tensorflow/stream_executor/cuda/cuda_blas.cc

```diff
@@ -101,18 +102,6 @@ static std::string ToString(cublasStatus_t status) {
   }
 }
 
-// Decide whether to enable TENSOR_OP_MATH
-static bool TensorOpMathEnabled() {
```

Comment:
State in the PR description that the environmental variable `TF_DISABLE_CUBLAS_TENSOR_OP_MATH` has been removed.

Reply:
Done.